docsrc/markdown/health_service.md

```diff
@@ -0,0 +1,25 @@
+# Health service
+The Health Service is a component at the service layer of the BDK which covers the Health Service part of the [REST API documentation](https://developers.symphony.com/restapi/reference).
+More precisely:
+* [Health check](https://developers.symphony.com/restapi/reference/health-check-v3)
+* [Health check extended](https://developers.symphony.com/restapi/reference/health-check-extended-v3)
+* [Agent info](https://developers.symphony.com/restapi/reference/agent-info-v1)
+
+
+## How to use
+The central component for the Health Service is the `UserService` class, it exposes the service APIs endpoints mentioned above.  
+The service is accessible from the`SymphonyBdk` object by calling the `users()` method:
```

Comment:
if i understand well, there is a mistake here,  for _the central component for Health service is the "**UserService**"_. 
same for the second line, we can call bdk object "**_users()_**" to access to the health service?